distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -307,6 +307,10 @@ grant codeBase "file:/admin-core-jolokia/javax.servlet-api/org.apache.shiro.core
     permission java.security.SecurityPermission "insertProvider";
 }
 
+grant codeBase "file:/security-filter-csrf" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}custom.system.properties", "read";
+}
```

Comment:
❓ Can't seem to figure out where the CSRF filter reads from `custom.system.properties`.

Reply:
In `etc/custom.system.properties` you can configure:
```
# Whether or not CSRF filtering should be enabled. This should only be disabled for testing and
# debugging purposes, otherwise the system will be left insecure.
csrf.enabled=true

# Comma separated list of host/port combinations that can be trusted to make requests to DDF.
csrf.trustedAuthorities=
```

Reply:
Ahhh ya I see that; thanks!

I just find it weird that a call to `System.getProperty` would require reading from `custom.system.properties`. I thought that some other bundle is responsible for reading entries from `custom.system.properties` and loading those entries into the system properties, and the CSRF filter would just be reading the system properties without ever having to even know about the `custom.system.properties` file 🤷 .